src/Seaborn.jl

```diff
@@ -53,706 +11,115 @@ function __init__()
-* `size_order` : list, optional
-    Specified order for appearance of the size variable levels, otherwise they are determined from the data. Not relevant when the size variable is numeric.
-* `size_norm` : tuple or Normalize object, optional
-    Normalization in data units for scaling plot objects when the size variable is numeric.
-* `legend` : “brief”, “full”, or False, optional
-    How to draw the legend. If “brief”, numeric hue and size variables will be represented with a sample of evenly spaced values. If “full”, every group will get an entry in the legend. If False, no legend data is added and no legend is drawn.
-* `kind` : string, optional
-    Kind of plot to draw, corresponding to a seaborn relational plot. Options are {scatter and line}.
-* `height` : scalar, optional
-    Height (in inches) of each facet. See also: aspect.
-* `aspect` : scalar, optional
-    Aspect ratio of each facet, so that aspect * height gives the width of each facet in inches.
-* `facet_kws` : dict, optional
-    Dictionary of other keyword arguments to pass to FacetGrid.
-* `kwargs` : key, value pairings
-    Other keyword arguments are passed through to the underlying plotting function.
-
-# Returns:	
-
-`g` : `FacetGrid`
-    Returns the FacetGrid object with the plot on it for further tweaking.
-"""
-
-function relplot(args...; kwargs...)
-    seaborn.relplot(args...; kwargs...)
+#metaprogram to export and define API functions
+# https://seaborn.pydata.org/api.html
+for func ∈ (
+    :relplot,
+    :scatterplot,
+    :lineplot,
+    :displot,
+    :histplot,
+    :kdeplot,
+    :ecdfplot,
+    :rugplot,
+    :catplot,
+    :stripplot,
+    :swarmplot,
+    :boxplot,
+    :violinplot,
+    :boxenplot,
+    :pointplot,
+    :barplot,
+    :countplot,
+    :lmplot,
+    :regplot,
+    :residplot,
+    :heatmap,
+    :clustermap,
+    :FacetGrid,
+    :pairplot,
+    :PairGrid,
+    :jointplot,
+    :JointGrid,
+    :set_theme,
+    :axes_style,
+    :set_style,
+    :plotting_context,
+    :set_context,
+    :set_color_codes,
+    :reset_defaults,
+    :reset_orig,
+    :set_palette,
+    :color_palette,
+    :husl_palette,
+    :hls_palette,
+    :cubehelix_palette,
+    :dark_palette,
+    :light_palette,
+    :diverging_palette,
+    :blend_palette,
+    :xkcd_palette,
+    :crayon_palette,
+    :mpl_palette,
+    :choose_colorbrewer_palette,
+    :choose_cubehelix_palette,
+    :choose_light_palette,
+    :choose_dark_palette,
+    :choose_diverging_palette,
+    :despine,
+    :move_legend,
+    :saturate,
+    :desaturate,
+    :set_hls_values,
+    :get_dataset_names,
+    :get_data_home
+)
+    s = string(func)
+    #export and define the function
+    @eval begin
+        export $func
+        @doc LazyHelp(seaborn, $s) function $func(args...; kwargs...)
+            seaborn.$func(args...; kwargs...)
```

Comment:
```suggestion
            seaborn.$s(args...; kwargs...)
```

Reply:
Why is it preferable to interpolate with the strings instead of symbols? I didn't understand your comment about auto-conversion.

Reply:
thanks for reviewing and helping again!